### PR TITLE
Require Node.js 6 and cleanup

### PR DESCRIPTION
**Pull request by SamVerschueren**
Fixed all the linting errors with the latest XO. Browser tests are failing though, still have to find what causes this. Also added a `.npmrc` file.

### CODE REVIEW

**Comment by SamVerschueren** [comment]
Believe I fixed the browser tests. It was just that AVA was not doing any assertions, so added a `t.notThrows` around `execa` for the browser tests.

**Comment by SamVerschueren** [comment]
Darn, will have to revert the const and let stuff as it targets Node 10 :).

**Comment by sindresorhus** [comment]
> Darn, will have to revert the const and let stuff as it targets Node 10 :).

Huh? Do you mean Node.js 4 and not 10? I plan to target Node.js 6 here, so not worry about breaking tests for lower than 6.

**Comment by SamVerschueren** [comment]
It still tests on 0.10 https://github.com/sindresorhus/any-observable/blob/master/.travis.yml. Maybe because of the browser stuff?

**Comment by sindresorhus** [comment]
I dunno. Feel free to remove the Node.js <6 stuff.

File: browser/create-karma-conf.js [hunk 1]
```diff
@@ -13,7 +13,13 @@ module.exports = function (files) {
...
 				debug: true
 			},
-			browsers: [process.env.CI ? 'PhantomJS' : 'Chrome'],
+			customLaunchers: {
+				ChromeHeadlessNoSandbox: {
+					base: 'ChromeHeadless',
+					flags: ['--no-sandbox']
+				}
+			},
+			browsers: [process.env.CI ? 'ChromeHeadlessNoSandbox' : 'ChromeHeadless'],
```
Comment (SamVerschueren) [inline browser/create-karma-conf.js:22]: The `--no-sandbox` is necessary for Travis CI https://docs.travis-ci.com/user/chrome.

File: package.json [hunk 2]
```diff
@@ -38,11 +38,9 @@
     "execa": "^0.4.0",
     "karma": "^1.1.0",
     "karma-browserify": "^5.0.4",
-    "karma-chrome-launcher": "^1.0.1",
+    "karma-chrome-launcher": "^2.2.0",
```
Comment (SamVerschueren) [inline package.json]: If you want, I can upgrade the other dependencies as well.

**Review by SamVerschueren — COMMENTED** [review]
Not sure if I tweaked everything regarding ES2016ifying the codebase. So feel free to give some feedback.

File: package.json (hunk 2 above)
Comment (sindresorhus) [reply package.json]: 👍

Comment (SamVerschueren) [reply package.json]: Done

**Comment by sindresorhus** [comment]
Thanks for the cleanup :)

**Pull request merged**